### ISSUE TITLE
Instead of topic name screenreader read json content in list

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [UnigramDev/Unigram#2914]
Hello everybody. In unigram appeared new accessibility issue. When some chat have at least one topic or more topics,in item with topics nvda and other screenreader announce things like this:
```
forumTopic {
  info = forumTopicInfo {
    message_thread_id = 1048576
    name = "General"
    icon = forumTopicIcon {
      color = 7322096
      custom_emoji_id = 0
    }
    creation_date = 1584115676
    creator_id = messageSenderChat {
      chat_id = -1001365288037
    }
    is_general = true
    is_outgoing = false
    is_closed = false
    is_hidden = false
  }
  last_message = message {
    id = 8143241216
    sender_id = messageSenderUser {
      user_id = 883824153
    }
    chat_id = -1001365288037
    sending_state = null
    scheduling_state = null
    is_outgoing = false
    is_pinned = false
    can_be_edited = false
    can_be_forwarded = true
    can_be_saved = true
    can_be_deleted_only_for_self = false
    can_be_deleted_for_all_users = false
    can_get_added_reactions = false
    can_get_statistics = false
    can_get_message_thread = true
```

Please fix this issue. Also if chat have at least one topic,when we press enter,cursor focused on button "Nothing to see here, tab further",but it should be focused on topics list.

**Comment by AlexandrKozlovskiy** [comment]
@FrayxRulez Thank you very much,now all works correct. Please do the same for stories. Because it very easy to do this,so in this case all or most of accessibility issues,which was reported at bugs.telegram.org will be fixed.
